src/client.test.jsx

```diff
@@ -47,27 +47,27 @@ describe('Client', () => {
     resetWindowValue('location', windowLocation);
   });
 
-  it('should hydrate client once routes are ready', async () => {
+  it('should hydrateRoot client once routes are ready', async () => {
```

Comment:
Find and replace error? 😸 
```suggestion
  it('should hydrate client once routes are ready', async () => {
```